Kind walkthrough update (#1938)
tektoncd/dashboard: docs/walkthrough/walkthrough-kind.md

The `kind-walkthrough` is the context for the cluster you created and the `*` at the beginning means that it is the currently active context.

> If you do not see `*` for `kind-walkthrough` that means its not currently active you can run 
> below command to select it and make it as active cluster 

Reviewer: We've already explained in the previous line that the `*` indicates the active context so no need to repeat that here, we can simplify it:

```suggestion
If it's not currently active you can run the following command to select it as the active cluster:
```

The Tekton Dashboard can import resources directly from a GitHub repository.

> Below steps will help you to understand how to import an example pipelinerun file [output-pipelinerun.yaml](https://github.com/tektoncd/pipeline/blob/master/examples/v1beta1/pipelineruns/output-pipelinerun.yaml)
> The dashboard will import this pipelinerun resources, which has a pipeline and that pipeline has some tasks that executes some steps sequentially. To learn more about how the pipelinerun works you can take a look at the example file [output-pipelinerun.yaml](https://github.com/tektoncd/pipeline/blob/master/examples/v1beta1/pipelineruns/output-pipelinerun.yaml).

Reviewer: We can simplify this a bit and avoid duplicating the link. Also where possible we should provide meaningful text as the content of the link instead of using the raw URL.

```suggestion
The following steps demonstrate how to import a set of resources from a file hosted in a GitHub repository and view the resulting `PipelineRun`. The example contains some `Tasks`, a `Pipeline` that uses them, and a `PipelineRun` to execute it. To learn more about what the example does, take a look at the [example file](https://github.com/tektoncd/pipeline/blob/master/examples/v1beta1/pipelineruns/output-pipelinerun.yaml) we'll be using.
```
